### PR TITLE
fix feedstock

### PR DESCRIPTION
**Pull request by carlodri**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

~__Do not merge yet!__ - waiting for feedback from the gods of conda...~ 😉

### CODE REVIEW

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Comment by carlodri** [comment]
@conda-forge/core I think I need some help on this, can someone please step in?

**Comment by carlodri** [comment]
Note to self: the latter is not a good permanent solution (breaks with a re-render). Should probably use `run_conda_forge_build_setup_win.bat` (and fix conda-smithy), WIP...

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Comment by carlodri** [comment]
@isuruf do you agree with my solution here (using `run_conda_forge_build_setup_win.bat`) until https://github.com/conda-forge/conda-smithy/pull/934 is merged?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,7 +50,7 @@ about:
   home: https://github.com/kennethreitz/responder/
   license: APACHE-2.0
   license_family: APACHE
-  license_file: LICENSE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (dhirschfeld) [inline recipe/meta.yaml]: IIUC this is unnecessary as more recent conda builds check this directory by default

Comment (carlodri) [reply recipe/meta.yaml]: sure, let's try a suggested change 😉

Comment (dhirschfeld) [reply recipe/meta.yaml]: Touché!

Comment (dhirschfeld) [inline recipe/meta.yaml]:
```suggestion
  license_file: LICENSE
```

Comment (carlodri) [reply recipe/meta.yaml]: looks like AppVeyor is not happy with suggested changes, had to rebuild the PR manually...

**Comment by carlodri** [comment]
LGTM, shall we merge or wait for a conda god blessing? I'm not entirely sure about this `run_conda_forge_build_setup_win.bat` thing... no docs, just inferred it from the `conda-smithy` source...

but I vote to merge anyway... if you agree

File: recipe/meta.yaml (hunk 1 above)
Comment (dhirschfeld) [reply recipe/meta.yaml]: Yeah, saw that - wasn't sure what was going on.

Other than that "suggested changes" seem pretty awesome!

I'm a big fan of the [papercuts](https://blog.github.com/2018-08-28-announcing-paper-cuts/) project.

Comment (carlodri) [reply recipe/meta.yaml]: looks cool!

**Comment by dhirschfeld** [comment]
Sorry - playing with the GitHub UI there. Just wanted to indicate that it LGTM and I'm personally happy for it to be merged.

I'm in no rush though so am happy for one of the conda gods to give their blessing first (since I don't feel qualified myself to comment on the particulars of your solution)

**Comment by carlodri** [comment]
> I'm in no rush though so am happy for one of the conda gods to give their blessing first...

me neither, so let's wait for @isuruf to give us a green light...

thanks

File: .appveyor.yml [hunk 2]
```diff
@@ -42,7 +42,8 @@ install:
 
     # Configure the VM.
     - cmd: conda.exe install -n root --quiet --yes conda-forge-ci-setup=2
-    - cmd: run_conda_forge_build_setup
+    # Overriding global run_conda_forge_build_setup_win with local copy.
```
Comment (mariusvniekerk) [inline .appveyor.yml:45]: Why?

Comment (carlodri) [reply .appveyor.yml:45]: https://github.com/conda-forge/conda-smithy/issues/931

And we need to get a dependency from a custom label here.

Comment (carlodri) [reply .appveyor.yml:45]: @mariusvniekerk do you think we can merge this? If it's ok, I would prefer not to wait for a new release of conda-smithy. We will update the feedstock accordingly when the release happens.

**Comment by carlodri** [comment]
@dhirschfeld if you agree I'm merging this...if we make a mess, we'll cleanup later... 😉

**Comment by dhirschfeld** [comment]
Go for it!

**Comment by carlodri** [comment]
💣

**Comment by dhirschfeld** [comment]
Thanks @carlodri for all your work pushing this forward!

**Comment by carlodri** [comment]
you're welcome, it's fun to work on this stuff!

**Comment by carlodri** [comment]
🎉 we got green lights everywhere in `master`! 🍾

**Pull request merged**